src/motioncore/utility/typedefs.h

```diff
@@ -188,4 +188,180 @@ enum class GarbledCircuitRole : unsigned int {
   kInvalid = 2  // for checking whether the value is valid
 };
 
+enum class UnsignedIntegerOperationType : unsigned int {
```

Comment:
is this actually still needed?

Reply:
Maybe we should keep  `UnsignedIntegerOperationType` and later create `SignedIntegerOperationType`. 
This would be helpful as operation identifiers in benchmarks.